client/src/components/Card.js

```diff
@@ -114,9 +120,11 @@ export default function Card({ title, price, symbol, stock }) {
         }),
       });
 
-      if (response.status !== 200) {
+      if (!response.ok) {
         throw new Error(response.error);
       }
+
+      getUserCardData();
```

Comment:
I would expect that `get` returns something.

```suggestion
      updateUserCardData();
```